bigquery-storage/api/BigQueryStorage.Samples/Data/CustomerRecord.cs

```diff
@@ -0,0 +1,232 @@
+// <auto-generated>
```

Comment:
I'd rename (or regenerate this) to use .g.cs as the file extension. When generating with protoc, use:

```sh
--csharp_opt=file_extension=.g.cs
```